types/rollup-plugin-visualizer/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for rollup-plugin-visualizer 2.6
+// Project: https://www.npmjs.com/package/rollup-plugin-visualizer
+// Definitions by: Max Boguslavskiy <https://github.com/maxbogus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="node" />
+import { Plugin } from 'rollup';
+
+export interface PluginVisualizerOptions {
+    filename?: string;
+    title?: string;
+    sourcemap?: boolean;
+    open?: boolean;
+    template?: string;
+    bundlesRelative?: boolean;
+}
+
+export default function visualizer(options?: PluginVisualizerOptions): Plugin;
```

Comment:
It seems like the [underlying library](https://unpkg.com/rollup-plugin-visualizer@2.6.0/plugin/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the "module-function" template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

Reply:
Agree. I reworked package according to your comments.